configuration/test/rao-runner-itools-config.yml

```diff
@@ -0,0 +1,10 @@
+rao:
```

Comment:
This config file is never used and there is a default file on gridcapa-rao-runner. Is it possible to delete it?